media-tv/v4l-dvb-saa716x/v4l-dvb-saa716x-0.0.1_p20170225-r5.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-info linux-mod
+
+DESCRIPTION="driver for saa716x based dvb cards like TT S2-6400 or Skystar 2 eXpress HD"
+HOMEPAGE="https://bitbucket.org/powARman/v4l-dvb-saa716x"
+
+REVISION="83f3bfd93a95"
+REVISION_DATE="20160322"
+
+SRC_URI="https://bitbucket.org/powARman/v4l-dvb-saa716x/get/${REVISION}.tar.bz2
+-> v4l-dvb-saa716x-0.0.1_p${REVISION_DATE}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+firmware"
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	firmware? ( sys-firmware/tt-s2-6400-firmware )"
```

Comment:
Looks like this has no function in this ebuild, then please use `optfeature` instead.
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

Reply:
Here I do not really agree. `optfeature` is not the right way, because the driver does not work on the hardware without the firmware. so IMHO "very specific circumstances, such as a package being nonfunctional unless at least one of a set of optional runtime dependencies is installed." applies here. Or may be not, because it is only one useflag, not a set of useflags.
Printing only a postinstall message can alse easily slipped, and the the correlation between driver package name and the firmware package is not easy visible. 
So maybe it would be the best to turn this to 'RDEPEND=sys-firmware/tt-s2-6400-firmware' without a useflag, if you insist of removing it.

Reply:
Sure, depending on it unconditionally sounds better then.